x11-misc/adriconf/adriconf-2.4.ebuild

```diff
@@ -0,0 +1,81 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake desktop
+
+DESCRIPTION="Advanced DRI Configurator"
+HOMEPAGE="https://github.com/jlHertel/adriconf"
```

Comment:
This seems dead, as it's marked read-only. But you seems to use anyway the newer fork by freedesktop.org?